dspback/triggers.py

```diff
@@ -70,9 +70,12 @@ async def watch_submissions():
                 public_json_ld = retrieve_submission_json_ld(document)
 
                 if public_json_ld:
-                    await db["discovery"].find_one_and_replace(
-                        {"repository_identifier": public_json_ld["repository_identifier"]}, public_json_ld, upsert=True
-                    )
+                    record_status = public_json_ld.pop("creativeWorkStatus", "Public")
```

Comment:
This is already done in the retrieve_submission_json_ld function call on line 70.